doc/modules/model_evaluation.rst

```diff
@@ -435,7 +435,7 @@ where :math:`1(x)` is the `indicator function
   >>> accuracy_score(y_true, y_pred)
   0.5
   >>> accuracy_score(y_true, y_pred, normalize=False)
-  2
+  2.0
```

Comment:
This is impacted because we now call `float(...)` explicitly on the output of `accuracy_score` and `zero_one_loss` to return a Python scalar instead of 0-dim numpy object whose dtype can vary in hard to predict ways depending on the inputs.

Reply:
Note: the docstring of `accuracy_score` specifies that the return type is float, so this might be considered a bugfix. Not sure if we need a changelog entry for this.

Reply:
I guess no one has noticed this difference till now because the 0-dim numpy array is mostly indistinguishable from a float?

Reply:
Yes, unless it as an int `dtype` as was the case in this particular code snippet prior to this PR :)